.github/workflows/ci_workflow.yml

```diff
@@ -280,35 +344,47 @@ jobs:
   # ---------------------------------------------------------------------------
 
   macos:
-    name: 'macOS 10.15 AppleClang 11.0 
-      <config=${{ matrix.build-type }}, 
+    name: 'macOS 10.15 
+      <AppleClang 11.0, 
+       config=${{ matrix.build-type }}, 
        shared=${{ matrix.build-shared }}, 
        sse=${{ matrix.use-sse }}, 
+       cxx=${{ matrix.cxx-version }}, 
        python=${{ matrix.python-version }}, 
        docs=${{ matrix.build-docs }}>'
     runs-on: macos-10.15
     strategy:
       matrix:
-        build: [1, 2, 3]
+        build: [1, 2, 3, 4]
         include:
           - build: 1
             build-type: Release
             build-shared: 'ON'
             build-docs: 'ON'
             use-sse: 'ON'
+            cxx-version: 11
             python-version: 3.7
           - build: 2
             build-type: Debug
             build-shared: 'ON'
             build-docs: 'OFF'
             use-sse: 'ON'
+            cxx-version: 11
             python-version: 3.7
           - build: 3
             build-type: Release
             build-shared: 'OFF'
             build-docs: 'ON'
             use-sse: 'OFF'
+            cxx-version: 11
             python-version: 2.7
+          - build: 4
+            build-type: Release
+            build-shared: 'ON'
+            build-docs: 'ON'
```

Comment:
Build 1 already generates the docs so it could be OFF for build 3 & 4.

Reply:
Done.